lib/fuse_lowlevel.c

```diff
@@ -2379,6 +2386,30 @@ static void do_destroy(fuse_req_t req, fuse_ino_t nodeid, const void *inarg)
 	send_reply_ok(req, NULL, 0);
 }
 
+static void
+do_get_fmap(fuse_req_t req, fuse_ino_t nodeid, const void *inarg)
+{
+	struct fuse_session *se = req->se;
+	(void)inarg;
+
+	if (se->op.get_fmap)
+		se->op.get_fmap(req, nodeid);
+	else
+		fuse_reply_err(req, -EOPNOTSUPP);
+}
+
+static void
+do_get_daxdev(fuse_req_t req, fuse_ino_t nodeid, const void *inarg)
+{
+	struct fuse_session *se = req->se;
+	(void)inarg;
+
+	if (se->op.get_daxdev)
+		se->op.get_daxdev(req, nodeid); /* Use nodeid as daxdev_index */
```

Comment:
Just quickly scanning through it, no reply?

Reply:
If I understand the question... Replies for get_fmap() and get_daxdev() are handled in the fuse server, if it handles those messages.  https://github.com/cxl-micron-reskit/famfs/blob/c6bdc79929b9988dd2ee84d54d74c8af622ac50c/src/famfs_fused.c#L681

If this breaks from a norm, let me know and I'll try to be more normy ;)

Another question: although the reply payload for get_daxdev is fixed-size, the reply payload for get_fmap should be variable - but is currently fixed at a max. Can you point me to example code for a variable-sized reply payload? I kinda just fumbled around until I got this part working, and then focused on several much bigger challenges...

Thanks!

Reply:
Just confused because reply types are not documented, see the other comment I had just added.

Reply:
Regarding dynamic reply, maybe as `fuse_reply_buf`, `fuse_reply_data or `fuse_reply_iov?